content/docs/latest/setup/storage/lvm.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Use LVM on Flatcar Container Linux
+linktitle: Using LVM
+description: How to use LVM to extend a logical volume across multiple disks 
+weight: 30
+aliases:
+- ../../os/lvm
+- ../../clusters/management/lvm
+---
+
+LVM - Logical Volume Management - allows you to create logical volumes, for example to use multiple physical disks as
+one volume. This allows you to make the full use of all attached disks.
+
+Flatcar Linux has built-in support for LVM.
+This guide covers create logical volumes using LVM and using them.
+
+
+## Creating LVM
+
+There are two main ways to do this: create everything manually or properly use an ignition config. We will first cover
+the manual way to get a better grip of what's happening and then the proper way.
+
+### Manual
+
+You can find all volumes using the `lsblk` command. For exampe:
+
+```shell
+
+# lsblk
+NAME    MAJ:MIN RM   SIZE RO TYPE  MOUNTPOINTS
+loop3     7:3    0  68.3M  1 loop
+loop4     7:4    0  39.3M  1 loop
+loop5     7:5    0     4K  1 loop
+sda       8:0    0 223.6G  0 disk
+|-sda1    8:1    0   128M  0 part
+|-sda2    8:2    0     2M  0 part
+|-sda3    8:3    0     1G  0 part
+| `-usr 254:0    0  1016M  1 crypt /usr
+|-sda4    8:4    0     1G  0 part
+|-sda6    8:6    0   128M  0 part  /oem
+|-sda7    8:7    0    64M  0 part
+`-sda9    8:9    0 221.3G  0 part  /
+sdb       8:16   0 447.1G  0 disk
+sdc       8:32   0 447.1G  0 disk
+sdd       8:48   0 223.6G  0 disk
+
+```
+
+Now we know that we have `/dev/sda`, `/dev/sdb`, `/dev/sdc`, `/dev/sdd` available. However, we cannot use `/dev/sda` in
+this scenario, but we can work with the others. 
+
+
+```shell
+# pvcreate /dev/sda /dev/sdb /dev/sdc /dev/sdd
+  WARNING: Failed to connect to lvmetad. Falling back to device scanning.
+  Device /dev/sda excluded by a filter.
```

Comment:
The paragraph above says that we cannot use `/dev/sda`, but here we still pass it to the `pvcreate` command. While `/dev/sda` seems to be ignored by `pvcreate` in the end ("Device /dev/sda excluded by a filter."), it's still confusing. So maybe we shouldn't be passing it there?

```suggestion
# pvcreate /dev/sdb /dev/sdc /dev/sdd
  WARNING: Failed to connect to lvmetad. Falling back to device scanning.
```

Reply:
Yeah I wasn't sure about this one. On the one hand, indeed we can't use it, on the other hand, I wanted to somewhat show that the command works for the others and just fails for the unusable